ui/tests/integration/components/pki/page/pki-tidy-status-test.js

```diff
@@ -0,0 +1,121 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'ember-qunit';
+import { render } from '@ember/test-helpers';
+import { hbs } from 'ember-cli-htmlbars';
+import { setupEngine } from 'ember-engines/test-support';
+import { setupMirage } from 'ember-cli-mirage/test-support';
+
+const SELECTORS = {
+  hdsAlertTitle: '.hds-alert .hds-alert__title',
```

Comment:
I wonder if using class selectors here could cause problems later when the HDS banner is used more throughout the app? We've had banner-selector issues before... 🤔  Thought son adding a test selector to the hds component in the `pki-tidy-status.hbs` template?